test/functional/basic/gesture-e2e-specs.js

```diff
@@ -113,6 +114,15 @@ describe('XCUITestDriver - gestures', function () {
 
       await driver.elementByAccessibilityId('2').should.not.be.rejected;
     });
+    it('should swipe the table and the bottom cell\'s Y position should change accordingly', async () => {
```

Comment:
Not a material change, but an observation: if you use backticks you don't need to escape the apostrophe!